index.html

```diff
@@ -5,9 +5,51 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
 </head>
 <body>
+
+  <main>
```

Comment:
I suggest you change this to a `<header>` tag. Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.

Websites usually have the hierarchy of `<header><main><footer>`
in the `header`, you have the `<nav>` element and title, in the `main` you have the rest of the page and in the `footer` you have a contact form or social media links or copyright information.